modular_skyrat/modules/customization/modules/jobs/_job.dm

```diff
@@ -81,10 +81,10 @@
 	banned_quirks = list(SEC_RESTRICTED_QUIRKS, HEAD_RESTRICTED_QUIRKS)
 
 /datum/job/chief_medical_officer
-	banned_quirks = list(HEAD_RESTRICTED_QUIRKS)
+	banned_quirks = list(HEAD_RESTRICTED_QUIRKS, "Blind" = TRUE) // SKYRAT EDIT
 
 /datum/job/chief_engineer
-	banned_quirks = list(HEAD_RESTRICTED_QUIRKS, "Paraplegic" = TRUE)
+	banned_quirks = list(HEAD_RESTRICTED_QUIRKS, TECH_RESTRICTED_QUIRKS, "Paraplegic" = TRUE) // SKYRAT EDIT
```

Comment:
This one is a good change, they should've had that to begin with, that was likely an oversight(?)